router/package_request.go

```diff
@@ -126,6 +129,40 @@ type respondToPackageRequestArgs struct {
 // respond crafts an appropriate response for a package request, serializes the
 // aforesaid response and sends it back to the original client.
 func (pr *packageRequest) respond(args respondToPackageRequestArgs) error {
+	// Git requests must be redirected to depot.
+	if isGitRequest(pr.parts) {
+		// Join the repo URL with the subpath of this request.
+		redirectLocation := fmt.Sprintf(
+			depotRepoURLTemplate,
+			getRequestDomain(pr.req),
+			depot.BuildHashedRepoName(
+				pr.parts.author,
+				pr.parts.repo,
+				pr.matchedSHA),
+			pr.parts.subpath)
+
+		// Issue a permanent redirect.
+		http.Redirect(
+			args.res,
+			pr.req,
+			redirectLocation,
+			http.StatusMovedPermanently)
+
+		// Without blocking, count a packfile request as a package download.
```

Comment:
By `count a packfile request as a package download.` are talking about clone requests coming from the git client?